packages/number-field/src/NumberField.ts

```diff
@@ -318,6 +320,13 @@ export class NumberField extends TextfieldBase {
             : event.deltaY / Math.abs(event.deltaY);
         if (direction !== 0 && !isNaN(direction)) {
             this.stepBy(direction * (event.shiftKey ? this.stepModifier : 1));
+            // Dispatch a change event after no new `input` events are queued by `wheel` interactions for 100ms.
+            clearTimeout(this.queuedChangeEvent);
+            this.queuedChangeEvent = setTimeout(() => {
+                this.dispatchEvent(
+                    new Event('change', { bubbles: true, composed: true })
+                );
+            }, 100) as unknown as number;
```

Comment:
The comment above addresses this number, but just to present an alternative, we could also consider something like:

```ts
const CHANGE_DEBOUNCE_MS = 100;
```

Reply:
Kind of like _both_ as it allows me to export this and leverage it in the test as well...